orthw

```diff
@@ -42,7 +42,7 @@ ort_config_copyright_garbage_file="$configuration_home/copyright-garbage.yml"
 ort_config_custom_license_texts_dir="$configuration_home/custom-license-texts"
 ort_config_how_to_fix_text_provider_script="$configuration_home/how-to-fix-text-provider.kts"
 ort_config_license_classifications_file="$configuration_home/license-classifications.yml"
-ort_config_notice_templates_dir="$configuration_home/notice-templates"
+ort_config_notice_templates_dir="$configuration_home/text-templates"
```

Comment:
Note that https://github.com/oss-review-toolkit/ort-config/pull/98 only changes the file name, not the directory name. And even if it did, it should probably be "*plain*-text-templates" for consistency.

Reply:
It was done already by https://github.com/oss-review-toolkit/ort-config/pull/100. Added it to the commit message.

Reply:
hm, was proposed by @tsteenbe  and approved by @MarcelBochtler .
No strong opinion on my side.